lib/tasks/scheduler.rake

```diff
@@ -40,29 +40,65 @@ task :collect_zpids => :environment do
   puts "Missing ZPIDs for #{missing_zpids.reload.where(zpid: nil).count}"
 end
 
+NeighborhoodCsvHeaders = %w(
+  neighborhood_name
+  neighborhood_zip_code
+  house_street_address
+  house_city
+  house_state
+  house_zip_code
+  house_bedrooms
+  house_bathrooms
+  house_square_feet
+).freeze
+
+desc "Generate a csv to onboard a neighborhood"
+task generate_hood_onboarding_csv: :environment do |t|
+  require "csv"
+
+  filename = "#{DateTime.now.to_i}_onboard_neighborhood.csv"
+  CSV.open(filename, "w") { |csv| csv << NeighborhoodCsvHeaders }
+
+  Rails.logger.info "Created #{filename}"
+end
+
 desc 'Upload a CSV of a neighborhood'
-task :upload_neighborhood, [:neighborhood_name, :neighborhood_zip_code, :neighborhood_data] => :environment do |t, args|
+task :upload_neighborhood, [:hood_onboarding_csv_path] => :environment do |t, args|
   starting_count = House.count
 
-  Rails.logger.info <<~MSG.strip
-    There are #{starting_count} houses in the database.
-    You have passed in #{args.count} arguments, with these keys: #{args.keys}
-  MSG
+  csv_entries = CSV.read(args[:hood_onboarding_csv_path], headers: true).map(&:to_h)
+
+  if csv_entries.flat_map(&:keys).uniq != NeighborhoodCsvHeaders
```

Comment:
Is `NeighborhoodCsvHeaders` available in this context? Isn't the constant defined in another task?

Reply:
Yah, the constant is defined at the file level (not in a task).